articles/openshift/howto-deploy-java-eap-app.md

```diff
@@ -0,0 +1,384 @@
+
+1. The application is opened in your browser looking similar to the following image:
+
+    ![OpenShift application running](./media/howto-deploy-java-eap-app/application_running_openshift.png)
+
+1. The application shows you the name of the pod which has served the information. To verify the clustering capabilities you could add some Todos, change the filter to see all the records and mark some of them as completed. Then delete the pod name that appears on the application, and once deleted, create a new Todo on the same application window. You will see that the new Todo is added via an ajax request, and you will continue getting the same list. It means your view, which was stored on the HTTP Session, has not been reset to a new one when the response is served by a different pod, indeed you will see that the session ID has not changed. However, this time the response was served by a different pod. That verifies that your HTTP Session data is being replicated across all your pods.
+
+## Clean up resources
+
+### Delete the application
+
+If you only want to delete your application, you can open the OpenShift console and, at the developer view, navigate to the `Helm` menu option. On this menu, you will see all the Helm Chart releases installed on your cluster. 
+
+   ![OpenShift uninstall application](./media/howto-deploy-java-eap-app/console_uninstall_application.png)
+
+Locate the `eap-todo-list-demo` Helm Chart and at the end of the row, click on the tree dots to open the action contextual menu entry.
+
+Click on "Uninstal Helm Release" to remove the application. Notice the application config map is not part of the chart and you need to remove it separately if you no longer need it.
+
+Execute the following command if you want to delete the configmap that hold the application configuration:
+
+```bash
+todo-list (bootable-jar-openshift) $  oc delete cm/todo-list-cm
+configmap "todo-list-cm" deleted
+```
+
+### Delete the database
+
+If you only want to delete the database and the related objects, execute the following command:
+
+```bash
+todo-list (bootable-jar-openshift) $ oc delete all -l app=mssql2019
+replicationcontroller "mssqlserver-1" deleted
+service "mssqlserver" deleted
+deploymentconfig.apps.openshift.io "mssqlserver" deleted
+
+todo-list (bootable-jar-openshift) $  oc delete cm/mssqlserver-cm
+configmap "mssqlserver-cm" deleted
+```
+
+### Delete the OpenShift project
+
+You can also delete all the configuration created for this demo by deleting the `eap-demo` project. To do so, execute the following:
+
+```bash
+todo-list (bootable-jar-openshift) $ oc delete project eap-demo
+project.project.openshift.io "eap-demo" deleted
+```
+
+### Delete the ARO cluster
+
+Delete the ARO cluster by following the steps in [Tutorial: Delete an Azure Red Hat OpenShift 4 cluster](./tutorial-delete-cluster.md)
+
+## Next steps
+
+In this guide, you learned how to:
+> [!div class="checklist"]
```

Comment:
This `[!div class="checklist"]` is being rendered - is it expected or should the line and the following line be removed?

Reply:
sorry, I should have mentioned before, to correctly display the final document, you need to do it by using Visual Studio Code with the Azure documentation tool package installed: https://marketplace.visualstudio.com/items?itemName=docsmsft.docs-authoring-pack

[!div class="checklist"] renders fine in Visual Studio with the doc pack installed.